testing/test_pdb.py

```diff
@@ -5017,6 +5042,13 @@ def test_truncate_to_visible_length(s, maxlength, expected):
 
 
 @pytest.mark.parametrize("pass_stdout", (True, False))
+@pytest.mark.skipif(
+    sys.platform == "win32",
+    reason=(
+        "IndexError: not a valid key: 'ck'. I think this is an issue"
+        " with pyreadline?"
+    ),
+)
```

Comment:
It uses `completekey="ck"`, which is wrong, but nobody complained.  Should use `tab` (the default) there then.

Reply:
Changing it to `tab` appears to have fixed things.